libraries/testkit/syncgateway.py

```diff
@@ -161,7 +169,10 @@ def restart(self, config, cluster_config=None):
             playbook_vars["revs_limit"] = '"revs_limit": {},'.format(revs_limit)
         except KeyError:
             log_info("revs_limit no found in {}, Ignoring".format(self.cluster_config))
+            playbook_vars["revs_limit"] = ''
 
+        if is_ipv6:
```

Comment:
same as above

Reply:
wrong implementation, but it would be true for IPv6 and IPv4 too. Fixing it.